### ISSUE TITLE
Support Dart output

### ISSUE THREAD

**Issue by daniel-v** [pageobject-io/pageobject-generator#1]
I am developing Angular 2 Dart applications and could use your tool to generate Page Objects for the templates.

There is already a tool out there that helps a lot:
https://github.com/google/pageloader/blob/master/dart/sync/PageLoaderManual.md

I believe, generating PageLoader compatible output would be the best way to go forward.

**Comment by szmeti** [comment]
Thanks for the great idea @daniel-v 

It would certainly add great value to the tool. Unfortunately, we have very limited resources at the moment and want to focus on having full Angular 1 & 2 support using languages like JavaScript and TypeScript first. 

On the other hand, we are open to contributions if you feel you could contribute to Angular 2 Dart support. We are going to provide some developer documentation for the project in the near future that should guide people who would like to contribute to the project.

**Comment by Slevender** [comment]
You mentioned TypeScript. When will there be Typescript support?

**Comment by szmeti** [comment]
At the moment we are porting the tool from Java to JavaScript. In the long term it will be a better language for the tool and will allow easier integration with other tools like angular-cli. Once this porting is done, adding Typescript support should be relatively straightforward.

The actual tests when written in TypeScript are not that radically different from JavaScript Protractor tests, so I guess it should not take too much time. I have created a new issue for TypeScript support #3
